### PR TITLE
Support: update generic 'email support' text

### LINKED ISSUE #54674: Support: update generic "email support" text
Since we have an email product, the generic use of ["email support" in Calypso](https://github.com/Automattic/wp-calypso/search?q=%22email+support%22), especially in Checkout's "included with your purchase" sidebar, to describe non-live chat support can be confusing for users.

Let's update it mentions of "email support" to "customer support via email".

/cc @olaolusoga @semihakocer @daledupreez

### PR DESCRIPTION
**Pull request by Patil2099**
Fixes #54674

### CODE REVIEW

File: client/components/marketing-survey/cancel-purchase-form/step-components/downgrade-step.jsx [hunk 1]
```diff
@@ -50,7 +50,7 @@ export class DowngradeStep extends Component {
 			refundReason = (
 				<p>
 					{ translate(
-						'WordPress.com Personal still gives you access to email support, removal of ads, and more — and for 50% of the cost of your current plan.'
+						'WordPress.com Personal still gives you access to customer support via email, removal of ads, and more — and for 50% of the cost of your current plan.'
```
Comment (a8ci18n) [inline client/components/marketing-survey/cancel-purchase-form/step-components/downgrade-step.jsx:53]: :ok: This change will be queued for retranslation. We'll use the existing translations in the meantime.

File: client/me/help/gm-closure-notice/index.jsx [hunk 2]
```diff
@@ -89,7 +89,7 @@ const GMClosureNotice = ( {
 	const MAIN_MESSAGES = {
 		before: {
 			hasPlan: translate(
-				'Live chat support will be closed from %(closes_at)s until %(reopens_at)s. Email support will remain open.',
+				'Live chat support will be closed from %(closes_at)s until %(reopens_at)s. customer support via email will remain open.',
```
Comment (a8ci18n) [inline client/me/help/gm-closure-notice/index.jsx]: :ok: This change will be queued for retranslation. We'll use the existing translations in the meantime.

File: packages/calypso-products/src/plans-list.js [hunk 3]
```diff
@@ -231,12 +231,12 @@ const getPlanBloggerDetails = () => ( {
 	getStoreAudience: () => i18n.translate( 'Best for bloggers' ),
 	getDescription: () =>
 		i18n.translate(
-			'{{strong}}Best for bloggers:{{/strong}} Brand your blog with a custom .blog domain name, and remove all WordPress.com advertising. Receive additional storage space and email support.',
+			'{{strong}}Best for bloggers:{{/strong}} Brand your blog with a custom .blog domain name, and remove all WordPress.com advertising. Receive additional storage space and customer support via email.',
```
Comment (a8ci18n) [inline packages/calypso-products/src/plans-list.js:234]: :information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **17** times:
`translate( '{{strong}}Best for Bloggers:{{/strong}} Brand your blog with a custom .blog domain name, and remove all WordPress.com advertising. Receive additional storage space and email support.' )` *ES Score: 9*
See 1 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/pull/54675)

File: packages/calypso-products/src/plans-list.js [hunk 4]
```diff
@@ -231,12 +231,12 @@ const getPlanBloggerDetails = () => ( {
...
 			plansDescriptionHeadingComponent
 		),
 	getShortDescription: () =>
 		i18n.translate(
-			'Brand your blog with a custom .blog domain name, and remove all WordPress.com advertising. Receive additional storage space and email support.'
+			'Brand your blog with a custom .blog domain name, and remove all WordPress.com advertising. Receive additional storage space and customer support via email.'
```
Comment (a8ci18n) [inline packages/calypso-products/src/plans-list.js:239]: :warning: This change will be queued for retranslation. We'll display the English text until then.

File: client/me/help/gm-closure-notice/index.jsx (hunk 2 above)
Comment (Aurorum) [inline client/me/help/gm-closure-notice/index.jsx]: It'd probably be best to preserve the capital letter here! :) 

```suggestion
				'Live chat support will be closed from %(closes_at)s until %(reopens_at)s. Customer support via email will remain open.',
```

**Review by jsnajdr — APPROVED** [review]
Looks good. As the new copy is longer, I was worried that it wouldn't fit well into certain space-constrained places in the UI, like the Plans grid. But trying it out shown me that it's fine and that the UI still looks good:

Screenshot 2021-07-19 at 16 22 15

File: client/me/help/gm-closure-notice/index.jsx (hunk 2 above)
Comment (michaeldcain) [reply client/me/help/gm-closure-notice/index.jsx]: Just noting that this is a blocker.

**Review by michaeldcain — CHANGES_REQUESTED** [review]

**Comment by Patil2099** [comment]
What should I change there @michaeldcain? Capital C  in customer ?

File: client/me/help/gm-closure-notice/index.jsx [hunk 5]
```diff
@@ -89,7 +89,7 @@ const GMClosureNotice = ( {
 	const MAIN_MESSAGES = {
 		before: {
 			hasPlan: translate(
-				'Live chat support will be closed from %(closes_at)s until %(reopens_at)s. Email support will remain open.',
+				'Live chat support will be closed from %(closes_at)s until %(reopens_at)s. Customer support via email will remain open.',
```
Comment (a8ci18n) [inline client/me/help/gm-closure-notice/index.jsx:92]: :ok: This change will be queued for retranslation. We'll use the existing translations in the meantime.

File: client/my-sites/checkout/composite-checkout/components/wp-checkout-order-summary.js [hunk 6]
```diff
@@ -195,10 +195,10 @@ function SupportText( { hasPlanInCart, isJetpackNotAtomic } ) {
 	const translate = useTranslate();
 
 	if ( hasPlanInCart && ! isJetpackNotAtomic ) {
-		return <span>{ translate( 'Unlimited email support' ) }</span>;
+		return <span>{ translate( 'Unlimited customer support via email' ) }</span>;
 	}
 
-	return <span>{ translate( 'Email support' ) }</span>;
+	return <span>{ translate( 'customer support via email' ) }</span>;
```
Comment (sirbrillig) [inline client/my-sites/checkout/composite-checkout/components/wp-checkout-order-summary.js]: Minor: I wonder if this should also be capitalised? Unless you lowercased it intentionally.

Comment (michaeldcain) [reply client/my-sites/checkout/composite-checkout/components/wp-checkout-order-summary.js]: @Patil2099: if you can make this update, I think this PR will be good to go. Thanks!

**Comment by Patil2099** [comment]
I think it is ready now @michaeldcain

**Review by sirbrillig — APPROVED** [review]
LGTM!

**Comment by michaeldcain** [comment]
@Patil2099: thanks again for the contribution! I've added the `[Status] String Freeze` label to queue this for translations so that we don't show English text in non-EN locales. Once the strings are retranslated, we'll merge this.

**Pull request merged**